### PR TITLE
Add None as acceptable for Optional fields.

### PR DESCRIPTION
**Pull request by e-agbanyo**
Using voluptuous' `Any()` to added `None` to product documents passed in by `detox`. 

These are the two services that would be affected by this and need to be updated:
https://github.com/iheartradio/objectify/search?q=pipeline.schema&unscoped_q=pipeline.schema
https://github.com/iheartradio/persist/search?q=pipeline.schema&unscoped_q=pipeline.schema

I pull the latest branches for these two services, pointed their requirements to this branch, pip  installed, and tox tested. They both passed without issues.

### CODE REVIEW

**Comment by ajbeach2** [comment]
Actually: does this have tests we need to update?

**Comment by lbedner** [comment]
Could we get some new test deliveries for these changes for our unit tests?

**Comment by lbedner** [comment]
Also, are we going to cut a new version of the pipeline so we can stop using github hashes in our requirement files?

**Comment by e-agbanyo** [comment]
A summary of all the changes made to pipeline have been offers related, bumping up the version of python, cleaning up docstrings, and now finally schema changes. This should be a bump up to either  1.2.0 or 1.5.0 or a jump to 2.0.0.

**Comment by zackmorris** [comment]
Just to be super nit-picky, on L#297 and L#298 we do `Any(None, <type>)`, but all of these changes are `Any(<type>, None)` - the order doesn't matter to me, just the consistency in order.

File: pipeline/schema.py [hunk 1]
```diff
@@ -291,7 +291,7 @@ def validate_schema(schema, message, logger=None):
 offer = SchemaAllRequired({
     'commercialModelType': CommercialModelType,
     'licensee': str,
-    Optional('price'): str,
+    Optional('price'): Any(str, None),
     'territoryCode': str,
     'useType': UseType,
     'validFrom': Any(None, OffsetAwareDatetime),
```
Comment (zackmorris) [inline pipeline/schema.py]: See [this](https://github.com/iheartradio/pipeline/pull/80#issuecomment-402813976) comment.

**Comment by e-agbanyo** [comment]
I am going to jump pipeline to `2.0.0` after this change is merged.

**Pull request merged**